src/Command/Validators.php

```diff
@@ -37,17 +31,13 @@ public static function validateUsername($username)
     }
 
     /**
-     * @static
-     *
-     * @param string $shortcut
-     *
      * @throws \InvalidArgumentException
      *
      * @return string[]
      *
      * @phpstan-return array{string, string}
      */
-    public static function validateEntityName($shortcut)
+    public static function validateEntityName(string $shortcut): array
```

Comment:
IMO, we must deprecate this method for some reasons:
1. It's using "entity" instead of "model";
2. It's using a model notation which is currently discouraged in favor of FQCNs;
3. It's not just validating, but returning a processed value.

Reply:
It's not used anymore. So yes Indeed we should